managed/services/agents/connection_checker.go

```diff
@@ -115,19 +115,11 @@ func (c *ConnectionChecker) CheckConnectionToService(ctx context.Context, q *ref
 			}
 		}
 	case models.PostgreSQLServiceType:
-		// TODO: remove the whole block after v3 release.
-		isSibSupported, err := isServiceInfoBrokerSupported(q, pmmAgentID)
-		if err != nil {
-			l.Warnf("Failed to check if serviceInfoBroker is supported: %s.", err)
-		}
-		// In newer clients this gets handled by the ServiceInfoBroker.
-		if !isSibSupported {
-			databaseCount := resp.(*agentpb.CheckConnectionResponse).GetStats().GetDatabaseCount() //nolint:forcetypeassert,staticcheck
-			agent.PostgreSQLOptions.DatabaseCount = databaseCount
-			l.Debugf("Updating database count: %d.", databaseCount)
-			if err = q.Update(agent); err != nil {
-				return errors.Wrap(err, "failed to update table count")
-			}
+		databaseCount := resp.(*agentpb.CheckConnectionResponse).GetStats().GetDatabaseCount() //nolint:forcetypeassert,staticcheck
```

Comment:
I meant that we don't need whole block in connection check

Reply:
I see. Removed completely from there.